src/components/Link.js

```diff
@@ -12,15 +12,14 @@ import { WrapperRow } from './Wrapper';
 export const Link = ({ url, description, openWebScreen }) => (
   <TouchableOpacity
     onPress={() => openLink(url, openWebScreen)}
-    accessibilityLabel={`${description} (Taste)`}
+    accessibilityLabel={`(${description}) Link`}
   >
     <WrapperRow>
       <Icon xml={link(colors.secondary)} style={styles.icon} />
       <RegularText primary>{description}</RegularText>
     </WrapperRow>
   </TouchableOpacity>
 );
-
```

Comment:
please keep the whitespace between these blocks of code.

Reply:
solved  f90ab7f07ff605af93121f7569894af606fd71e3

Reply:
i am not seeing a change related ti this in that commit either

Reply:
now solved f90ab7f